### ISSUE TITLE
Fable testing: searchable list issues

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#4849]
## Description
A number of issues arose in the Fable testing of the searchable list component:
- Content from the list is read unexpectedly: NVDA in Chrome was observed to read the complete content of the list without direction by the user. This made it hard for the user to make a selection.

https://app.makeitfable.com/share/675aee2bce532ae8996c2c4d

- VoiceOver in Safari does not announce the selections of the list once something is entered into the field.
- VoiceOver in Safari did not give the context required for the user to understand they needed to clear the list in order to see more options.

https://app.makeitfable.com/share/675ae863ce532ae8996c2c4c

**Comment by srtalbot** [comment]
@connorscarolyns  for testing.
